cli/admin/src/main/java/io/pravega/cli/admin/utils/ControllerHostnameVerifier.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.cli.admin.utils;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+
+public class ControllerHostnameVerifier implements HostnameVerifier {
```

Comment:
Curious, why do you need a custom hostname verifier here? You could use one of the standard ones. 

Here's how we do it in the client (for the call to segment store): 

https://github.com/pravega/pravega/blob/899a82d5c4055007311adf01bf58d1629f4e2f6e/client/src/main/java/io/pravega/client/connection/impl/TcpClientConnection.java#L243-L251

Reply:
Instead of having a `HostnameVerfier`, using `SSLContext` as suggested.